### PR TITLE
Add lucid package

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc @hhslepicka

### CODE REVIEW

File: recipes/lucid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  requires:
+    - pytest
+    - pytest-qt
+
+about:
+  home: https://github.com/pcdshub/lucid
+  summary: 'SLAC National Accelerator Laboratory: LCLS User Control and Interface Design'
+  license: SLAC Open
+  license_family: OTHER
+  license_file: LICENSE.txt
```
Comment (hhslepicka) [inline recipes/lucid/meta.yaml]: It should be `LICENSE.md`

**Comment by hhslepicka** [comment]
I accept to be a maintainer of this package

**Comment by npavlovikj** [comment]
The error message is:
```
conda.exceptions.ResolvePackageNotFound: 
  - typhos
```
so please add `typhos` to conda-forge first.

**Comment by npavlovikj** [comment]
Sorry, just saw this https://github.com/conda-forge/staged-recipes/pull/11060.

**Comment by hhslepicka** [comment]
@npavlovikj would you mind reviewing this one which is the last dependency of Typhos and will allow us to move forward with the dependencies for this package?
https://github.com/conda-forge/staged-recipes/pull/11065

**Comment by npavlovikj** [comment]
It usually takes around half an hour to an hour for the package to appear in the channel, so please try restarting the CI a bit later for `typhos`.

**Comment by hhslepicka** [comment]
@conda-forge-admin, please restart ci

File: recipes/lucid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - pip
+  run:
+    - python >=3.6
+    - fuzzywuzzy
+    - ophyd
+    - pyqt >=5
+    - pyqtads
+    - pyyaml
+    - qtpy
+    - typhos
```
Comment (hhslepicka) [inline recipes/lucid/meta.yaml]: We are missing happi

**Comment by hhslepicka** [comment]
@conda-forge/help-python this one is ready to go.

**Comment by npavlovikj** [comment]
Thank you @klauer !

**Comment by klauer** [comment]
Thanks for reviewing + merging!

**Pull request merged**